src/styles/_utils.scss

```diff
@@ -1,3 +1,4 @@
+@import "utils/reset";
```

Comment:
Importing reset and normalize.css in the same project might not be necessary as they both aim to create a consistent baseline style. Select one based on your requirements.